workspace-manager-clienttests/README.md

```diff
@@ -16,16 +16,25 @@ Note: `render-config.sh` assumes a valid `.vault-token` in your `$HOME` director
 ./render-config.sh
 ```
 
-To run the test, use the following command from the workspace-manager-clienttests directory.
+To run the tests, use one of the following commands from the workspace-manager-clienttests directory.
+NB: The `BasicIntegration.json` and `BasicNightlyPerf.json` are sample test suite configurations for Integration and Perf tests.
+See `test-runner-integration.yml` and `test-runner-nightly-perf.yml` in `.github/workflows` for use cases.
 
+Integration Test
 ```
-./gradlew  runTest --args="configs/BasicAuthenticated.json /tmp/TR"
+./gradlew  runTest --args="suites/BasicIntegration.json /tmp/TR"
 ```
 
-The default server that this test will run against is specified in the resources/configs/BasicAuthenticated.json file.
+Perf Test
+```
+./gradlew  runTest --args="suites/BasicPerf.json /tmp/TR"
+```
+
+The default server that the test will run against is specified in the resources/configs/integration/BasicAuthenticated.json file, using integration test as example.
```

Comment:
Nit: the default server is specified in the test config file.

Reply:
Updated comments.